lib/elixir/lib/inspect/algebra.ex

```diff
@@ -565,6 +600,10 @@ defmodule Inspect.Algebra do
     end
   end
 
+  defp ansi(color) do
+    color |> List.wrap |> IO.ANSI.format_fragment(true)
```

Comment:
I would prefer to wrap `precolor` and `postcolor` in the `color/3` function. The reason for that is that it is then guaranteed the color field in the `doc_color` tuple is always a list (i.e. let's do input sanitization when the data is created not when it is processed).